include/ccl_core.h

```diff
@@ -122,7 +122,7 @@ typedef struct ccl_data{
 } ccl_data;
 
 /**
- * Sturct containing references to instances of the above structs, and boolean flags of precomputed values.
+ * Struct containing references to instances of the above structs, and boolean flags of precomputed values.
  */
```

Comment:
Did you intend to add the new paramslist object to ccl_cosmology, or is it correct that only a comment changed here?

Reply:
I didn't intend to add it yet. That's the next step though.